File: native/mixdog-graph/tests/fixtures/calls/elixir/sample.ex
# hidden()
_quoted = "hidden()"

plain()

defmodule Widget do
  def run(a) do
    inner()
    nest(leaf(1))
    a.b().c()
  end

  def act() do
    helper()
    Foo.ping()
  end

  def new() do
    :ok
  end
end

mark = "μ"; seed()
Widget.new()

File: native/mixdog-graph/tests/fixtures/calls/r/sample.r
# hidden()
quoted <- "hidden()"

inner <- function() { invisible(NULL) }
leaf <- function(x) { x }
nest <- function(x) { x }
helper <- function() { invisible(NULL) }
seed <- function() { invisible(NULL) }
plain <- function() { invisible(NULL) }

plain()

run <- function(a) {
  inner()
  nest(leaf(1))
  a$b()$c()
}

act <- function(self) {
  helper()
  self$ping()
}

mark <- "μ"; seed()
Widget()
